src/main/java/com/accolite/pru/health/AuthApp/controller/AuthController.java

```diff
@@ -144,19 +149,22 @@ public ResponseEntity registerUser(@ApiParam(value = "The RegistrationRequest pa
      * the app itself.
      */
     @PostMapping("/password/resetlink")
-    @ApiOperation(value = "Receive the reset link request and publish event to send mail containing the password " +
+    @Operation(summary = "Receive the reset link request and publish event to send mail containing the password " +
             "reset link")
-    public ResponseEntity resetLink(@ApiParam(value = "The PasswordResetLinkRequest payload") @Valid @RequestBody PasswordResetLinkRequest passwordResetLinkRequest) {
+    public ResponseEntity resetLink(@Param(value = "The PasswordResetLinkRequest payload") @Valid @RequestBody PasswordResetLinkRequest passwordResetLinkRequest) {
 
         return authService.generatePasswordResetToken(passwordResetLinkRequest)
                 .map(passwordResetToken -> {
-                    UriComponentsBuilder urlBuilder = ServletUriComponentsBuilder.fromCurrentContextPath().path("/api/auth/password/reset");
-                    OnGenerateResetLinkEvent generateResetLinkMailEvent = new OnGenerateResetLinkEvent(passwordResetToken,
+                    UriComponentsBuilder urlBuilder = ServletUriComponentsBuilder.fromCurrentContextPath().path("/api" +
+                            "/auth/password/reset");
```

Comment:
You could probably have `/api` as a const

Reply:
I just follow what You have